com.avaloq.tools.ddk.xtext.export/src/com/avaloq/tools/ddk/xtext/export/generator/ResourceDescriptionStrategyGenerator.xtend

```diff
@@ -88,7 +88,7 @@ class ResourceDescriptionStrategyGenerator {
                     if («guard») {

```

Comment:
Should the block not be?

                  «IF c.guard === null»
                    «generateCaseBody(c, ctx, genModelUtil)»
                  «ELSIF !guard.equalsIgnoreCase("false")»
                    «javaContributorComment(c.guard.location)»
                    if («guard») {
                      «generateCaseBody(c, ctx, genModelUtil)»
                    }
                  «ENDIF»

this way the is no need to check twice if c.guard is null or not